example/Grunt-ReadMe.md

```diff
@@ -0,0 +1,46 @@
+# Grunt
+
+This template uses [Grunt](http://gruntjs.com/), the Javascript Task Runner. To use Grunt with this Template App, do the following:
+
+* Install grunt: ```npm install -g grunt-cli```
```

Comment:
Would recommend just adding `grunt` and `grunt-cli` to `package.json`. Better yet, maybe don't use grunt and instead use just npm scripts like so:

```
"start": "NODE_PATH=. nodemon application.js"
"lint": "eslint lib/*.js lib/**/*.js "
```

That's what we're doing typically. Grunt seems to confuse people new to node and new to the platform and the value its adds is pretty questionable IMHO.